<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Icosaedro</class>
 <widget class="QMainWindow" name="Icosaedro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dado - Icosaedro</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Proyecto2n/Recursos/img/logo.png</normaloff>Proyecto2n/Recursos/img/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(Proyecto2n/Recursos/img/fondoelemento.png);
background-repeat: no-repeat;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
background-repeat: no-repeat;
border-style: none;</string>
    </property>
    <property name="text">
     <string>Icosaedro</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pbico">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Proyecto2n/Recursos/img/ico/plantilla.png);
 border-style: none;
background-repeat: no-repeat;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnlanza">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>420</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
background-color: #555555;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Â¡Lanzalo!</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pb8">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Proyecto2n/Recursos/img/moneda/iniciop.png);
 border-style: none;
background-repeat: no-repeat;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
</string>
    </property>
    <property name="text">
     <string>Ultimos 16</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pb12">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Proyecto2n/Recursos/img/moneda/iniciop.png);
 border-style: none;
background-repeat: no-repeat;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pb7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Proyecto2n/Recursos/img/moneda/iniciop.png);
 border-style: none;
background-repeat: no-repeat;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pb15">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Proyecto2n/Recursos/img/moneda/iniciop.png);
 border-style: none;
background-repeat: no-repeat;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pb6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Proyecto2n/Recursos/img/moneda/iniciop.png);
 border-style: none;
background-repeat: no-repeat;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pb14">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Proyecto2n/Recursos/img/moneda/iniciop.png);
 border-style: none;
background-repeat: no-repeat;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pb16">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Proyecto2n/Recursos/img/moneda/iniciop.png);
 border-style: none;
background-repeat: no-repeat;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pb1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Proyecto2n/Recursos/img/moneda/iniciop.png);
 border-style: none;
background-repeat: no-repeat;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pb10">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Proyecto2n/Recursos/img/moneda/iniciop.png);
 border-style: none;
background-repeat: no-repeat;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Proyecto2n/Recursos/img/tapete.png);
background-repeat: no-repeat;
border-style: none;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pb2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Proyecto2n/Recursos/img/moneda/iniciop.png);
 border-style: none;
background-repeat: no-repeat;
</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pb5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Proyecto2n/Recursos/img/moneda/iniciop.png);
 border-style: none;
background-repeat: no-repeat;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pb11">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Proyecto2n/Recursos/img/moneda/iniciop.png);
 border-style: none;
background-repeat: no-repeat;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pb4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Proyecto2n/Recursos/img/moneda/iniciop.png);
 border-style: none;
background-repeat: no-repeat;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pb13">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Proyecto2n/Recursos/img/moneda/iniciop.png);
 border-style: none;
background-repeat: no-repeat;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pb9">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Proyecto2n/Recursos/img/moneda/iniciop.png);
 border-style: none;
background-repeat: no-repeat;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pb3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Proyecto2n/Recursos/img/moneda/iniciop.png);
 border-style: none;
background-repeat: no-repeat;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnautomatico">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>480</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
background-color: #555555;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Lanzalo por mi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnhome">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
background-color: #555555;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>MenÃº principal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnestadisticas">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
background-color: #555555;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>EstadÃ­sticas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnreiniciarbd">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
background-color: #555555;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Reinciar estadÃ­sticas</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>label</zorder>
   <zorder>pbico</zorder>
   <zorder>pb8</zorder>
   <zorder>label_2</zorder>
   <zorder>pb12</zorder>
   <zorder>pb7</zorder>
   <zorder>pb15</zorder>
   <zorder>pb6</zorder>
   <zorder>pb14</zorder>
   <zorder>pb16</zorder>
   <zorder>pb1</zorder>
   <zorder>pb10</zorder>
   <zorder>pb2</zorder>
   <zorder>pb5</zorder>
   <zorder>pb11</zorder>
   <zorder>pb4</zorder>
   <zorder>pb13</zorder>
   <zorder>pb9</zorder>
   <zorder>pb3</zorder>
   <zorder>btnhome</zorder>
   <zorder>btnlanza</zorder>
   <zorder>btnautomatico</zorder>
   <zorder>btnestadisticas</zorder>
   <zorder>btnreiniciarbd</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
